test/e2e/tests/test_tenant_health.py

```
@@ -0,0 +1,270 @@
+        "eof",
+        "temporary failure",
+        "network is unreachable",
+    ]
+    stderr_lower = stderr.lower()
+    return any(p in stderr_lower for p in patterns)
+
+
+def _run_oc(args, retries=3):
+    """Run an oc command with transient-error retry. Returns (stdout, stderr, returncode)."""
+    for attempt in range(retries):
+        result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)